From: Simon McVittie <smcv@collabora.com>
Date: Wed, 25 Mar 2020 13:03:47 +0000
Subject: common: Add variant-schema-compiler output to nodist SOURCES

This means it doesn't go into dist tarballs, and we don't need to add
it to BUILT_SOURCES and CLEANFILES separately because it's already
there.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://github.com/flatpak/flatpak/pull/3489
---
 common/Makefile.am.inc | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/common/Makefile.am.inc b/common/Makefile.am.inc
index d9779c4..f7d93ad 100644
--- a/common/Makefile.am.inc
+++ b/common/Makefile.am.inc
@@ -78,9 +78,6 @@ CLEANFILES += $(nodist_libflatpak_common_base_la_SOURCES)
 common/flatpak-variant-private.h common/flatpak-variant-impl-private.h: variant-schema-compiler/variant-schema-compiler  data/flatpak-variants.gv
 	$(AM_V_GEN) $(srcdir)/variant-schema-compiler/variant-schema-compiler --outfile-header common/flatpak-variant-private.h  --outfile common/flatpak-variant-impl-private.h --prefix var $(srcdir)/data/flatpak-variants.gv
 
-BUILT_SOURCES += common/flatpak-variant-private.h common/flatpak-variant-impl-private.h
-CLEANFILES += common/flatpak-variant-private.h common/flatpak-variant-impl-private.h
-
 libflatpak_common_base_la_SOURCES = \
 	common/flatpak-utils-base.c \
 	common/flatpak-utils-base-private.h \
@@ -101,6 +98,8 @@ nodist_libflatpak_common_la_SOURCES = \
 	$(systemd_dbus_built_sources)	\
 	$(xdp_dbus_built_sources) \
 	common/flatpak-enum-types.c \
+	common/flatpak-variant-private.h \
+	common/flatpak-variant-impl-private.h \
 	$(NULL)
 
 BUILT_SOURCES += $(nodist_libflatpak_common_la_SOURCES)
@@ -156,8 +155,6 @@ libflatpak_common_la_SOURCES = \
 	common/flatpak-utils-private.h \
 	common/flatpak-utils.c \
 	common/valgrind-private.h \
-	common/flatpak-variant-private.h \
-	common/flatpak-variant-impl-private.h \
 	$(NULL)
 
 if HAVE_LIBMALCONTENT
